concepticondata/references/references.bib

```diff
@@ -2981,6 +2981,16 @@ @article{Alario1999
   publisher={Springer}
 }
 
+@article{Saxena2013,
+  title={Carving Tibeto-Kanauri by its joints: Using basic vocabulary lists for genetic grouping of languages},
+  author={Saxena, Anju and Borin, Lars},
+  journal={Approaches to measuring linguistic differences},
```

Comment:
This was not a journal, but a book, so you have to change the bibtex to `@incollection`, with publisher, etc. Check for key `Saxena2013` at https://digling.org/evobib/ where you find a correct bibtex

Reply:
Thanks for the tip!